xml/admin_monitoring_alerting.xml

```diff
@@ -41,72 +40,37 @@
  </itemizedlist>
  <para>
   The &prometheus; configuration and <emphasis>scrape</emphasis> targets
-  (exporting daemons) are setup automatically by &deepsea;. &deepsea; also
+  (exporting daemons) are setup automatically by &cephadm;. &cephadm; also
   deploys a list of default alerts, for example <literal>health
   error</literal>, <literal>10% OSDs down</literal>, or <literal>pgs
   inactive</literal>.
  </para>
 
- <sect1 xml:id="pillar-variables">
-  <title>Pillar Variables</title>
-  <para>The &spillar; is a key-value store that provides information and
-    configuration values to minions. It is available to all minions, each
-    with differing content. The &spillar; is pre-populated with default
-    values and can be customized in two different ways:</para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      <emphasis role="bold"><filename>/srv/pillar/ceph/stack/global.yml</filename></emphasis>:
-      to change pillar variables for all nodes.
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      <emphasis role="bold"><filename>/srv/pillar/ceph/stack/<replaceable>CLUSTER_NAME</replaceable>/minions/<replaceable>HOST</replaceable></filename></emphasis>:
-      to change specific minion configurations.
-     </para>
-    </listitem>
-   </itemizedlist>
-  <para>
-    The pillar variables below are available to all nodes by default:
-  </para>
+ <sect1 xml:id="monitoring-cephadm-config">
+   <title>Configuring Monitoring and Alerting</title>
 <screen>
-  monitoring:
-  alertmanager:
-    config: salt://path/to/config
-    additional_flags: ''
-  grafana:
-    ssl_cert: False # self-signed certs are created by default
-    ssl_key: False # self-signed certs are created by default
-  prometheus:
-    # pass additional configration to prometheus
-    additional_flags: ''
-    alert_relabel_config: []
-    rule_files: []
-    # per exporter config variables
-    scrape_interval:
-      ceph: 10
-      node_exporter: 10
-      prometheus: 10
-      grafana: 10
-    relabel_config:
-      alertmanager: []
-      ceph: []
-      node_exporter: []
-      prometheus: []
-      grafana: []
-    metric_relabel_config:
-      ceph: []
-      node_exporter: []
-      prometheus: []
-      grafana: []
-    target_partition:
-      ceph: '1/1'
-      node_exporter: '1/1'
-      prometheus: '1/1'
-      grafana: '1/1'
+  ---
```

Comment:
We'll need to document how this configuration file can be used, otherwise the user will get stuck here, not knowing what to do.

The content should be in a file `monitoring.yaml` (name can freely be chosen). Then `ceph orch apply -i monitoring.yaml` would deploy the configured services.

Reply:
A configuration file like the following can be used to deploy all services using cephadm, not only the ones from monitoring. How that works is also explained elsewhere and it might make sense to provide a link for the user, so that they can read up the details elsewhere and we may be able to keep the explanation here a little bit more concise.

The grey at the bottom of this chapter: https://ceph.readthedocs.io/en/latest/cephadm/install/#deploy-additional-monitors-optional

Note that "deploying additional monitors" is completely unrelated to "monitoring". "Monitors are a service that is a core component of Ceph, whereas "monitoring" means additional software is used to monitor the health of a cluster (and their hosts, so even outside of Ceph).

Reply:
Using the configuration file to deploy the monitoring stack is just one of two ways to do it. The other way is using the CLI. This is also documented elsewhere (at least [upstream](https://docs.ceph.com/docs/master/cephadm/monitoring/#deploying-monitoring-with-cephadm)). Both ways work the same way, the difference is just how it's done.